src/styles/sass/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  padding: 16px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  font-size: $base-font-size;
+  font-weight: 400;
+  border-radius: 5px;
+
+  @include smooth-animation;
+
+  &__img-container {
+    margin-top: 16px;
+    padding: 0 3px;
```

Comment:
you may center it using flex (align-items) or margin-auto (horizontally) instead of 3px padding